File: Models/Korisnik.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;

namespace Models{
    public class Korisnik{

        [Key]
        public int Id { get; set; }
        
        [Required]
        public string Ime{get; set;}

        [Required]
        public string Prezime{get; set;}

        [Required]
        public string Username{get; set;}

        [Required]
        [MinLength(8)]
        public string Sifra{get;set;}

        [JsonIgnore]
        public List<Rezervacija> Rezervacije{get; set;}

    }
}

File: Client/Korisnik.js
import { Salon } from "./Salon.js";

export class Korisnik {
    constructor(ime,prezime,username,sifra,salon) {
        this.ime = ime;
        this.prezime = prezime;
        this.username = username;
        this.sifra = sifra;
        this.kont = null;
        this.salon = salon;
    }
   
}